navigation/create_control/config/create_2_control.yaml

```
@@ -0,0 +1,7 @@
+create1:
+  mobile_base_controller:
+    type: "diff_drive_controller/DiffDriveController"
+    left_wheel: 'wheel_right_joint'
+    right_wheel: 'wheel_left_joint'
+    pose_covariance_diagonal: [0.001, 0.001, 1000000.0, 1000000.0, 1000000.0, 1000.0]
+    twist_covariance_diagonal: [0.001, 0.001, 1000000.0, 1000000.0, 1000000.0, 1000.0]
```

`create1` is too specific for one robot. And also is not needed.

```suggestion
mobile_base_controller:
  type: "diff_drive_controller/DiffDriveController"
  left_wheel: 'wheel_right_joint'
  right_wheel: 'wheel_left_joint'
  pose_covariance_diagonal: [0.001, 0.001, 1000000.0, 1000000.0, 1000000.0, 1000.0]
  twist_covariance_diagonal: [0.001, 0.001, 1000000.0, 1000000.0, 1000000.0, 1000.0]
```

You can easily debug it running this:

```
GUI=false RVIZ=True ROS_CONTROL=True roslaunch ca_gazebo create_house.launch
```